.gitignore

```diff
@@ -4,6 +4,7 @@ jank-generated.hpp
 a.out
 .jank-native-repl-history
 .jank-repl-history
+**/.envrc
```

Comment:
No need for `**/` here. By default, a name on its own will match in any directory.